modules/docs/mdx/MAINTAINING.mdx

```diff
@@ -20,6 +20,8 @@ Kit!
   - [Forward Merge Workflow](#forward-merge-workflow)
   - [Release Minor Workflow](#release-minor-workflow)
   - [Release Workflow](#release-workflow)
+- [Deprecations](#deprecations)
+	- [How To Deprecate](#how-to-deprecate)
```

Comment:
@mannycarrera4 Thanks for putting this PR together! I'm still reviewing the bulk of the PR, but for now, can we move the `Deprecations` section to the end of the document (below `Codemods`)? The `GitHub Actions` and `Releases` sections are closely related so I'd like to keep them together instead of having `Deprecations` sit in between.

We'll need to move the `Deprecations` item in the table of contents as well.